app/main.py

```diff
@@ -1,6 +1,29 @@
+import os
+
+import requests
+
+
+URL = "http://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
+API_KEY = os.environ.get("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(
+        f"{URL}?key={API_KEY}&q={FILTERING}&aqi=no")
```

Comment:
fix indentation here, also there is no need to use f-string here, u can use params arg instead